app/Locale/German.php

```diff
@@ -111,6 +124,8 @@ public function get_locale(): array
 			'IMPORT_SERVER' => 'Von Server importieren',
 			'NEW_ALBUM' => 'Neues Album',
 			'NEW_TAG_ALBUM' => 'Neues Tag-Album',
+			'UPLOAD_TRACK' => 'Upload track',
+			'DELETE_TRACK' => 'Delete track',
```

Comment:
```suggestion
			'UPLOAD_TRACK' => 'Track hochladen',
			'DELETE_TRACK' => 'Track löschen',
```

I am German ;-)

Reply:
Me too 🙂, but I wasn't sure if I should do it in this PR. I'll translate them in my future  PRs directly.